README.md

```diff
@@ -16,80 +16,166 @@ These steps are detailed in the [Google Cloud documentation](https://cloud.googl
+```
 
-2. Create Wagtail's database tables and an initial user:
+### Install the Cloud SQL proxy
+
+This is necessary so Wagtail can connect to its database to run migrations and 
+other operations on the local machine.
+
+On 64-bit Linux:
 
 ```
-./manage.py migrate
-./manage.py createsuperuser
+curl -Lo cloud_sql_proxy https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 
+chmod 755 cloud_sql_proxy
 ```
 
-## Check everything works locally
+For other platforms, see the
+[https://cloud.google.com/python/django/flexible-environment#install_the_sql_proxy](GCP documentation)
+for download links.
 
-`./manage.py runserver`
+Enable application default credentials in the GCP SDK:
+
+```
+gcloud auth application-default login
+```
+
+Fetch the instance connection name for the database you created:
+
+```
+gcloud sql instances describe wagae --format='value(connection_name)'
+torchkube:europe-west2:wagae
+```
+
+Replace `[CONNECTION_NAME]` with the connection name.
+
+## Create Google Storage buckets
+
+GCS will be used to store static assets and uploaded media.  Create two new
+buckets, using a common prefix:
+
+```
+gsutil mb gs://my-wagtail-site-static
+gsutil mb gs://my-wagtail-site-media
+```
 
-The styling will be broken, but you should be able to log in to the Wagtail admin interface at http://127.0.0.1:8000/admin/ with the user details you have just created.
+Configure CORS headers on the static bucket so fonts can be loaded from the
+site's domain, and make the contents of both buckets readable by default:
 
-## Create a bucket for static content
+```
+gsutil cors set cors.json gs://my-wagtail-site-static
+gsutil defacl set public-read gs://my-wagtail-site-static
+gsutil defacl set public-read gs://my-wagtail-site-media
+```
+
+## Configure local settings
+
+Create a file called `wagae/settings/local.py`, and add your site-specific
+settings:
 
 ```
-gsutil mb gs://<GCS BUCKET NAME>
-gsutil defacl set public-read gs://<GCS BUCKET NAME>
-gsutil cors set cors.json gs://<GCS BUCKET NAME>
+DB_CONNECTION_NAME = '[CONNECTION_NAME]'
+DB_PASSWORD = '[DB_PASSWORD]'
+GS_BUCKET_PREFIX = '[BUCKET_PREFIX]'
+GS_PROJECT_ID = '[GCP_PROJECT_ID]'
+SECRET_KEY = '[SECRET_KEY]'
 ```
 
-Update `STATIC_URL` in `wagae/settings/base.py`:
+`[GCP_PROJECT_ID]` is the name of your GCP project.  `[BUCKET_PREFIX]` is the
+common prefix of the two GCS buckets you created, e.g. `my-wagtail-site`.
+
+`[SECRET_KEY]` should be a long string of random bytes, e.g. generated by
+`pwgen -s 64 1`.
+
+## Install the database tables
 
-```python
-STATIC_URL = 'https://storage.googleapis.com/<GCS BUCKET NAME>/static/'
+1. Run Cloud SQL Proxy in a separate terminal:
+
+`./cloud_sql_proxy -instances="[CONNECTION_NAME]=tcp:5432"`
+
+2. Create Wagtail's database tables and an initial user:
+
+```
+./manage.py migrate
+./manage.py createsuperuser
 ```
 
-and copy the files across:
+## Upload static files
+
+Ideally, we would let `django-storages` upload static files for us, but
```

Comment:
I don't think we need this aside

Reply:
Okay, then I propose changing it to:

```
Because we didn't configure GCS for Django's static file storage, we need to
collect the static assets locally, then manually upload them to the GCS bucket:
```